CHANGELOG.md

```diff
@@ -111,6 +111,7 @@
 * [BUGFIX] Dashboards: fix "Mimir / Rollout Progress" latency comparison when gateway is enabled. #10495
 * [BUGFIX] Dashboards: fix autoscaling panels when Mimir is deployed using Helm. #10473
 * [BUGFIX] Alerts: fix `MimirAutoscalerNotActive` alert. #10564
+* [BUGFIX] Dashboards: fix "Mimir / Tenants" legends for baremetal deployments.
```

Comment:
@ldvc could you rebase the PR on top of latest `main` branch, since Mimir 2.16 release has been just cut.

Also, please add the PR reference to the change-line:

```suggestion
* [BUGFIX] Dashboards: fix "Mimir / Tenants" legends for non-Kubernetes deployments. #10891
```

Reply:
Thanks for the suggestions. I've rebased the changes, should be good now.